### PR TITLE
docs: new Dependabot TRG

### PR DESCRIPTION
**Pull request by tomaszbarwicki**
PR to introduce new TRG for Dependabot. It's a GitHub feature which supports security and versions' updates automation. 
Idea behind the TRG creation is to help the project keep up to date in best, optimal effort wise way.

Proposing to park it as next item in the GIT section, any suggestions for better place always welcomed.

Fixes https://github.com/eclipse-tractusx/sig-infra/issues/150

### CODE REVIEW

File: docs/release/trg-0/trg-2-6.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+Dependabot is an excellent fit for application dependencies/vulnerabilities. By regularly checking for updates, it allows you to seamlessly integrate the latest improvements into your application.
+
+For Docker images, Dependabot ensures that your base images and dependencies are regularly updated, reducing the risk of using outdated or vulnerable components.
+
+Dependabot can also assist in keeping used GitHub Actions up to date. This is crucial for ensuring that your workflows leverage the latest GitHub Actions features and improvements.
+
+### Security updates
+
+To enable Dependabot for security updates, you can leverage GitHub's Security tab. Go to the "Security" tab in your repository and follow the prompts to enable automated security updates.
```
Comment (SebastianBezold) [inline docs/release/trg-0/trg-2-6.md:29]: Is this something we can do via UI, or do we need to do that via changes to the Otterdog config, so an org admin account can trigger these changes?

**Review by SebastianBezold — COMMENTED** [review]
I would add a "caution" section, mentioning the `DEPENDENCIES` file and Dash.
Just merging Dependabot PRs can easily lead to an out-of-date `DEPENDENCIES` file.

File: docs/release/trg-0/trg-2-6.md (hunk 1 above)
Comment (tomaszbarwicki) [reply docs/release/trg-0/trg-2-6.md:29]: I think you can actually do it two ways. I was able to enable it in UI and also noticed that some repositories have it enabled via Otterdog config, using below flag:
 dependabot_security_updates_enabled: true

**Comment by tomaszbarwicki** [comment]
> I would add a "caution" section, mentioning the `DEPENDENCIES` file and Dash. Just merging Dependabot PRs can easily lead to an out-of-date `DEPENDENCIES` file.

Added https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/588/commits/a8fa4273838abbc885b85c1e4fbf6c190ed3e9dc

File: docs/release/trg-0/trg-2-6.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+- Security: Receive timely updates for security vulnerabilities in your project's dependencies.
+- Stability: Keep your project stable by staying current with the latest releases.
+- Efficiency: Automate the time consuming task of manually checking for updates and creating pull requests.
+
+## Description
+
+Dependabot is an excellent fit for application dependencies/vulnerabilities. By regularly checking for updates, it allows you to seamlessly integrate the latest improvements into your application.
+
+For Docker images, Dependabot ensures that your base images and dependencies are regularly updated, reducing the risk of using outdated or vulnerable components.
```
Comment (FaGru3n) [inline docs/release/trg-0/trg-2-6.md:23]: Please add a reference to the Docker image tags and versioning TRG

File: docs/release/trg-0/trg-2-6.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 5
+```
+
+More information:  
+<https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates>  
+<https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file>
+
+:::caution
```
Comment (FaGru3n) [inline docs/release/trg-0/trg-2-6.md:70]: I would guess not all reading to the end of the hole TRG maybe you can add the "Caution" section a bit higher in this TRG.

**Review by FaGru3n — CHANGES_REQUESTED** [review]
some hints and tipps

File: docs/release/trg-0/trg-2-6.md (hunk 2 above)
Comment (tomaszbarwicki) [reply docs/release/trg-0/trg-2-6.md]: https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/588/commits/cc352d9561ba5ef7c887aee4ff3b826c77ff201d

File: docs/release/trg-0/trg-2-6.md (hunk 3 above)
Comment (tomaszbarwicki) [reply docs/release/trg-0/trg-2-6.md]: https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/588/commits/cc352d9561ba5ef7c887aee4ff3b826c77ff201d

**Review by SebastianBezold — APPROVED** [review]
👍  Will you announce it on the mailing list first?
At least for the API versioning TRG proposal, the feedback was, that discussing on a PR is easier, than discussing on the draft section

**Review by FaGru3n — APPROVED** [review]
LGTM

**Pull request merged**